### PR TITLE
PMM-7983 fix OVF job

### PR DESCRIPTION
**Pull request by yurkovychv**
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ovf-image-test/detail/pmm2-ovf-image-test/2221/pipeline

### CODE REVIEW

File: pmm/pmm2-ovf-test.groovy [hunk 1]
```diff
@@ -58,7 +59,7 @@ pipeline {
             description: 'Postgre SQL Server version',
             name: 'PGSQL_VERSION')
         string(
-            defaultValue: '--addclient=ps,1 --addclient=mo,1',
+            defaultValue: '--addclient=haproxy,1 --addclient=ps,1',
```
Comment (puneet0191) [inline pmm/pmm2-ovf-test.groovy:62]: not using haproxy too, lets remove this from setup

File: pmm/pmm2-ovf-test.groovy [hunk 2]
```diff
@@ -1,10 +1,11 @@
-void runUITests(CLIENT_VERSION, CLIENT_INSTANCE, SERVER_IP, GIT_BRANCH) {
-    stagingJob = build job: 'pmm2-ui-tests', parameters: [
+void runUITests(CLIENT_VERSION, CLIENT_INSTANCE, SERVER_IP, GIT_BRANCH, CLIENTS) {
+    stagingJob = build job: 'pmm2-ui-tests-temp', parameters: [
```
Comment (puneet0191) [inline pmm/pmm2-ovf-test.groovy:2]: remember to change it back to normal and disable the temp job.

File: pmm/pmm2-ui-tests.groovy [hunk 3]
```diff
@@ -133,9 +133,6 @@ pipeline {
     options {
         skipDefaultCheckout()
     }
-    triggers {
-        upstream upstreamProjects: 'pmm2-server-autobuild', threshold: hudson.model.Result.SUCCESS
```
Comment (puneet0191) [inline pmm/pmm2-ui-tests.groovy:137]: bring this back after your final check

**Review by puneet0191 — CHANGES_REQUESTED** [review]

File: pmm/pmm2-ovf-test.groovy (hunk 1 above)
Comment (yurkovychv) [reply pmm/pmm2-ovf-test.groovy]: done, thanks

File: pmm/pmm2-ovf-test.groovy (hunk 2 above)
Comment (yurkovychv) [reply pmm/pmm2-ovf-test.groovy]: done

File: pmm/pmm2-ui-tests.groovy (hunk 3 above)
Comment (yurkovychv) [reply pmm/pmm2-ui-tests.groovy]: done

**Review by puneet0191 — APPROVED** [review]

**Pull request merged**